tools/etcdhelper/etcdhelper.go

```diff
@@ -11,7 +11,7 @@ import (
 	"time"
 
 	jsonserializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
-	api "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/kubectl/scheme"
```

Comment:
would have stayed with the legacyscheme. Remembering us resurrecting an old HPA GroupVersion which appears in our customers' storage, but is certainly not in the latest kubectl.